build-support/pants_venv

```diff
@@ -11,15 +11,19 @@ REQUIREMENTS=(
   "${REPO_ROOT}/3rdparty/python/requirements.txt"
 )
 
-# NB: We house these outside the working copy to avoid needing to gitignore them, but also to
-# dodge https://github.com/hashicorp/vagrant/issues/12057.
-platform=$(uname -mps | sed 's/ /./g')
-venv_dir_prefix="${HOME}/.cache/pants/pants_dev_deps/${platform}"
+platform=$(uname -mps)
```

Comment:
If this isn't actually used outside the function, then moving it inside could be good.

Reply:
Hm. Yea, fine I guess.